cli/src/main/java/com/box/l10n/mojito/cli/command/ThirdPartySyncCommand.java

```diff
@@ -38,6 +43,12 @@
     @Parameter(names = {"--project", "-p"}, arity = 1, required = true, description = "Third party project to synchronize with")
     String thirdPartyProjectId;
 
+    @Parameter(names = {"--actions", "-a"}, variableArity = true, required = false, description = "Actions to synchronize", converter = ThirdPartySyncActionsConverter.class)
+    List<ThirdPartySync.Action> actions = Arrays.asList(ThirdPartySync.Action.MAP_TEXTUNIT, ThirdPartySync.Action.PUSH_SCREENSHOT);
+
+    @Parameter(names = {"--options", "-o"}, variableArity = true, required = false, description = "Options to synchronize")
+    List<String> options = new ArrayList<>();
```

Comment:
nit pick, but for other list like "filterOptions" we don't have an empty array as default value. It makes the CLI doc look different. I'd rather remove it to have something more consistent.